src/server/api/routers/ckb.ts

```diff
@@ -1,30 +1,26 @@
+import BigNumber from 'bignumber.js'
 import { explorerService } from '../../../services/ExplorerService'
+import { nodeService } from '../../../services/NodeService'
 import { createTRPCRouter, publicProcedure } from '../trpc'
 
+const DECIMAL = BigNumber(10 ** 8)
```

Comment:
Not a good name, maybe it should be something like `ckbShannonPrecision` ?

Reply:
`DECIMAL` is a generally used term in blockchain context

Reply:
Oh, I see.